Sources/SwiftKafka/Configuration/KafkaProducerConfig.swift

```diff
@@ -59,6 +59,17 @@ public struct KafkaProducerConfig: Hashable, Equatable {
         set { self.dictionary["message.send.max.retries"] = String(newValue) }
     }
 
+    /// Allow automatic topic creation on the broker when subscribing to or assigning non-existent topics.
+    /// The broker must also be configured with auto.create.topics.enable=true for this configuration to take effect.
+    /// Note: the default value (true) for the producer is different from the default value (false) for the consumer.
+    /// Further, the consumer default value is different from the Java consumer (true), and this property
+    /// is not supported by the Java producer.
+    /// Requires broker version >= 0.11.0.0, for older broker versions only the broker configuration applies.
```

Comment:
The problem with the default value is that you are stating for the producer it is false here but if you query this property then it will evaluate to `true`. However at runtime if nothing is set RDKafka will use `false` for it.